### PR TITLE
Workflow Automated Test - LRQA-68834 | master

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasks.path [hunk 1]
```diff
@@ -133,6 +133,21 @@
...
+<tr>
+	<td>MY_WORKFLOW_TASKS_SEARCH_BUTTON</td>
+	<td>//button[contains(@type,'submit')]//*[contains(@class,'lexicon-icon-search')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MY_WORKFLOW_TASKS_SEARCH_RESULTS_BAR</td>
+	<td>//span[contains(.,'Results') and @class='text-truncate']</td>
+	<td></td>
+</tr>
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasks.path:150]: You can sync those paths created with other created by Paulo Alves at that [PR](https://github.com/rodrigocunhaa/liferay-portal/pull/823/files). 
 
Suggestion:

MY_WORKFLOW_TASKS_SEARCH_INPUT_FIELD
MY_WORKFLOW_TASKS_SEARCH_ICON
MY_WORKFLOW_TASKS_SEARCH_RESULT_BAR

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 2]
```diff
@@ -378,12 +378,36 @@ definition {
...
 
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		Type(
+			locator1 = "MyWorkflowTasks#MY_WORKFLOW_TASKS_SEARCH_BAR",
+			value1 = "Blogs Entry");
+
+		Click(locator1 = "MyWorkflowTasks#MY_WORKFLOW_TASKS_SEARCH_BUTTON");
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:406]: You can take that piece and create a macro to make search at MyWorkflowTask area, see a example at [PR](https://github.com/rodrigocunhaa/liferay-portal/pull/823/files).

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Review by lucasperj — APPROVED** [review]

**Comment by lucasperj** [comment]
Just started reviewing :)

**Comment by lucasperj** [comment]
Combined and sent on [PR#3](https://github.com/lucasperj/liferay-portal/pull/3). Thank you!

**Pull request closed without merging**